src/DssSpell.sol

```diff
@@ -18,185 +18,401 @@ pragma solidity 0.8.16;
+}
+
+interface RwaUrnLike {
+    function lock(uint256 wad) external;
+    function hope(address usr) external;
+}
+
+interface RwaInputConduitLike {
+    function mate(address usr) external;
+    function file(bytes32 what, address data) external;
+}
+
+interface RwaOutputConduitLike {
+    function file(bytes32 what, address data) external;
+    function hope(address usr) external;
+    function mate(address usr) external;
+    function kiss(address who) external;
+}
+
+interface VestLike {
+    function restrict(uint256 _id) external;
+    function create(address _usr, uint256 _tot, uint256 _bgn, uint256 _tau, uint256 _eta, address _mgr) external returns (uint256 id);
+    function yank(uint256 _id) external;
+}
+interface NetworkPaymentAdapterLike {
+    function bufferMax() external view returns (uint256);
+    function minimumPayment() external view returns (uint256);
+    function file(bytes32 what, uint256 data) external;
+    function file(bytes32 what, address data) external;
 }
 
-interface GemLike {
-    function transfer(address, uint256) external returns (bool);
+interface DssCronSequencerLike {
+    function windows(bytes32) external view returns (uint256 start, uint256 length);
+}
+
+interface PoolConfiguratorLike {
+    struct InitReserveInput {
+        address aTokenImpl;
+        address stableDebtTokenImpl;
+        address variableDebtTokenImpl;
+        uint8 underlyingAssetDecimals;
+        address interestRateStrategyAddress;
+        address underlyingAsset;
+        address treasury;
+        address incentivesController;
+        string aTokenName;
+        string aTokenSymbol;
+        string variableDebtTokenName;
+        string variableDebtTokenSymbol;
+        string stableDebtTokenName;
+        string stableDebtTokenSymbol;
+        bytes params;
+    }
+    function initReserves(InitReserveInput[] calldata input) external;
+    function configureReserveAsCollateral(
+        address asset,
+        uint256 ltv,
+        uint256 liquidationThreshold,
+        uint256 liquidationBonus
+    ) external;
+    function setBorrowableInIsolation(address asset, bool borrowable) external;
+    function setDebtCeiling(address asset, uint256 newDebtCeiling) external;
+}
+
+interface AaveOracleLike {
+    function setAssetSources(address[] calldata assets, address[] calldata sources) external;
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/8f548f10a4ce1db0acdc30fb171eebb72b236c39/governance/votes/Executive%20Vote%20-%20May%2017%2C%202023.md' -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/ce40e721ba58dc631ee1b66f5259423dd8e504ce/governance/votes/Executive%20vote%20-%20May%2024%2C%202023.md' -q -O - 2>/dev/null)"
```

Comment:
I got commit `812543c7c79b09e7cad6b1ff77e965fe2e84acd2` instead of the one currently used (`ce40e721ba58dc631ee1b66f5259423dd8e504ce`) so we should discuss this. Files and content are identical, this discrepancy happens but I want to note how we go about getting the two different hashes.